### PR TITLE
Add benchs to Asset

### PR DESCRIPTION
**Pull request by fmiguelgarcia**
Benchmarks for Asset pallets.

I've also added some **length constraints** to some _unbounded types_:
- Asset name, limited by `AssetNameMaxLenght`(default = 128),
- Name of the funding round, limited by `FundingRoundNameMaxLength` (default = 128) 

Other stuff:
- Use of `schnorrkell` from `cryptography` library, which removes compilation errors and simplifies our dependency management.
- Remove `set_proposal_cool_off_period` from benchmarks. I was removed from its pallet on a previous comment.

### CODE REVIEW

File: pallets/asset/src/benchmarking.rs [hunk 1]
```diff
@@ -76,130 +143,326 @@ benchmarks! {
...
+        let new_owner = UserBuilder::<T>::default().build_with_did("new_owner", SEED);
+        let ticker = make_ticker::<T>(owner.origin().into());
+
+        Module::<T>::asset_ownership_relation(owner.did(), ticker.clone());
+        let new_owner_auth_id = identity::Module::<T>::add_auth(
+            owner.did(),
+            Signatory::from(new_owner.did()),
+            AuthorizationData::TransferTicker(ticker),
+            None,
+            );
```
Comment (maxsam4) [inline pallets/asset/src/benchmarking.rs:162]: indentation

File: pallets/weights/src/pallet_asset.rs [hunk 2]
```diff
@@ -1,24 +1,130 @@
...
-	// WARNING! Some components were not used: ["f"]
-	fn create_asset(n: u32, i: u32, ) -> Weight {
-		(96128000 as Weight)
-			.saturating_add((35000 as Weight).saturating_mul(n as Weight))
-			.saturating_add((74000 as Weight).saturating_mul(i as Weight))
-			.saturating_add(DbWeight::get().reads(12 as Weight))
-			.saturating_add(DbWeight::get().writes(11 as Weight))
-	}
+    // WARNING! Some components were not used: ["t"]
+    fn register_ticker() -> Weight {
```
Comment (satyamakgec) [inline pallets/weights/src/pallet_asset.rs:11]: I think need to bump up the `t` value to see the effect

**Review by satyamakgec — APPROVED** [review]

File: pallets/weights/src/pallet_asset.rs (hunk 2 above)
Comment (fmiguelgarcia) [reply pallets/weights/src/pallet_asset.rs:10]: I've removed `t` directly because TICKER_LEN is fixed to 12. We don't need it.

**Review by maxsam4 — APPROVED** [review]
The CI needs to be pleased but LGTM

**Review by maxsam4 — APPROVED** [review]

**Review by satyamakgec — APPROVED** [review]

**Pull request merged**